pkg/cinder/dbsync.go

```diff
@@ -55,16 +57,16 @@ func DbSyncJob(instance *cinderv1beta1.Cinder, labels map[string]string) *batchv
 								RunAsUser: &runAsUser,
 							},
 							Env:          env.MergeEnvs([]corev1.EnvVar{}, envVars),
-							VolumeMounts: GetVolumeMounts(false),
+							VolumeMounts: GetVolumeMounts(false, dbSyncExtraMounts, DbsyncPropagation),
 						},
 					},
-					Volumes: GetVolumes(instance.Name, false),
+					Volumes: GetVolumes(instance.Name, false, dbSyncExtraMounts, DbsyncPropagation),
 				},
 			},
 		},
 	}
 
-	job.Spec.Template.Spec.Volumes = GetVolumes(ServiceName, false)
+	job.Spec.Template.Spec.Volumes = GetVolumes(ServiceName, false, dbSyncExtraMounts, DbsyncPropagation)
```

Comment:
I just realized, I think, that this line is unnecessary given that we already have it in L63 (or vice-versa).  63 uses `instance.Name` and this line uses `ServiceName` as the first arg, though.  Not sure off the top of my head which one is the right choice.